examples/discord_bot.py

```diff
@@ -31,15 +31,15 @@
 @coc.ClanEvents.member_join(tags=clan_tags)
 async def on_clan_member_join(member, clan):
     await bot.get_channel(INFO_CHANNEL_ID).send(
-        "{0.name} ({0.tag}) just " "joined our clan {1.name} ({1.tag})!".format(member, clan)
+        f"{member.name} ({member.tag}) just " "joined our clan {clan.name} ({clan.tag})!"
     )
 
 
 @coc_client.event
 @coc.ClanEvents.member_name(tags=clan_tags)
 async def member_name_change(old_player, new_player):
     await bot.get_channel(INFO_CHANNEL_ID).send(
-        "Name Change! {0.name} is now called {1.name} (his tag is {1.tag})".format(old_player, new_player)
+        f"Name Change! {old_player.name} is now called {new_player.name} (his tag is {new_player.tag})"
```

Comment:
While we're here would you mind changing `his` to the gender neutral `their`, please?